src/ontology/mondo-edit.obo

```diff
@@ -270747,7 +270751,7 @@ xref: UMLS:C3693482 {source="MEDGEN:811326", source="MONDO:equivalentTo", source
 is_a: MONDO:0000653 {source="MONDO:Redundant", source="MONDO:indirect"} ! integumentary system cancer
 is_a: MONDO:0005164 {source="DOID:3507"} ! fibrosarcoma
 relationship: excluded_subClassOf MONDO:0019300 {source="Orphanet:31112", source="https://orcid.org/0000-0001-5208-3432"} ! obsolete rare skin tumor or hamartoma
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/8800 {source="MONDO:mim2gene_medgen", source="OMIM:607907"} ! PDGFB
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/8800 {source="MONDO:mim2gene_medgen"} ! PDGFB
```

Comment:
Based on [OMIM:607907](https://omim.org/entry/607907) it seems this gene association should also be removed.

Reply:
Evidence was removed, but the `mondo` pipeline likely needs to be updated now (see: [thoughts](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1933016180)).

Reply:
Ok, so the OMIM evidence was removed since this has a mapping key of 4 and does not meet the guidelines for disease defining, but due to the gene association also having `source="MONDO:mim2gene_medgen"` and that the OMIM pipeline would not have removed the association even if there were no source provenance this gene association remains.